openedx/features/marketplace/constants.py

```diff
@@ -1,36 +1,100 @@
+ORGANIZATIONAL_SECTOR_ART_AND_CULTURE = 'art-and-culture'
+ORGANIZATIONAL_SECTOR_OTHER_KEY = 'other-sector'
+
 ORGANIZATION_SECTOR_CHOICES = (
-    ('health-and-well-being', _('Health & Well-being')),
-    ('education', _('Education')),
-    ('gender-equality', _('Gender Equality')),
-    ('sanitation', _('Sanitation')),
-    ('climate-changes', _('Climate Change')),
-    ('clean-energy', _('Clean Energy')),
-    ('environmental-conservation', _('Environmental Conservation')),
-    ('work-and-economic-growth', _('Work & Economic Growth')),
-    ('human-rights', _('Human Rights')),
-    ('social-justice', _('Social Justice')),
-    ('art-and-culture', _('Art & Culture'))
+    (ORGANIZATIONAL_SECTOR_HEALTH_AND_WELL_BEING, _('Health & Well-being')),
+    (ORGANIZATIONAL_SECTOR_EDUCATION, _('Education')),
+    (ORGANIZATIONAL_SECTOR_GENDER_EQUALITY, _('Gender Equality')),
+    (ORGANIZATIONAL_SECTOR_SANITATION, _('Sanitation')),
+    (ORGANIZATIONAL_SECTOR_CLIMATE_CHANGES, _('Climate Change')),
+    (ORGANIZATIONAL_SECTOR_CLEAN_ENERGY, _('Clean Energy')),
+    (ORGANIZATIONAL_SECTOR_ENVIRONMENTAL_CONSERVATION, _('Environmental Conservation')),
+    (ORGANIZATIONAL_SECTOR_WORK_AND_ECONOMIC_GROWTH, _('Work & Economic Growth')),
+    (ORGANIZATIONAL_SECTOR_HUMAN_RIGHTS, _('Human Rights')),
+    (ORGANIZATIONAL_SECTOR_SOCIAL_JUSTICE, _('Social Justice')),
+    (ORGANIZATIONAL_SECTOR_ART_AND_CULTURE, _('Art & Culture'))
 )
 
-ORGANIZATIONAL_PROBLEM_CHOICES = (
-    ('healthcare-supplies', _('Healthcare Supplies')),
-    ('remote-working-tools', _('Remote Working Tools')),
-    ('online-training', _('Online Training')),
-    ('delivery-services', _('Delivery Services')),
-    ('mentorship', _('Mentorship')),
-    ('funding', _('Funding')),
-    ('human-resources', _('Human Resources')),
+ORGANIZATION_SECTOR_CHOICES_VIEW = (
+    (ORGANIZATIONAL_SECTOR_HEALTH_AND_WELL_BEING, _('Health & Well-being')),
+    (ORGANIZATIONAL_SECTOR_EDUCATION, _('Education')),
+    (ORGANIZATIONAL_SECTOR_GENDER_EQUALITY, _('Gender Equality')),
+    (ORGANIZATIONAL_SECTOR_SANITATION, _('Sanitation')),
+    (ORGANIZATIONAL_SECTOR_CLIMATE_CHANGES, _('Climate Change')),
+    (ORGANIZATIONAL_SECTOR_CLEAN_ENERGY, _('Clean Energy')),
+    (ORGANIZATIONAL_SECTOR_ENVIRONMENTAL_CONSERVATION, _('Environmental Conservation')),
+    (ORGANIZATIONAL_SECTOR_WORK_AND_ECONOMIC_GROWTH, _('Work & Economic Growth')),
+    (ORGANIZATIONAL_SECTOR_HUMAN_RIGHTS, _('Human Rights')),
+    (ORGANIZATIONAL_SECTOR_SOCIAL_JUSTICE, _('Social Justice')),
+    (ORGANIZATIONAL_SECTOR_ART_AND_CULTURE, _('Art & Culture')),
+    (ORGANIZATIONAL_SECTOR_OTHER_KEY, 'Other'),
 )
 
+
+
+USER_SERVICES_DELIVERY_SERVICES_KEY = 'delivery-services'
+USER_SERVICES_FUNDING_KEY = 'funding'
+USER_SERVICES_HEALTH_CARE_SUPPLIES_KEY = 'healthcare-supplies'
+USER_SERVICES_MENTORSHIP_KEY = 'mentorship'
+USER_SERVICES_ONLINE_TRAINING_KEY = 'online-training'
+USER_SERVICES_PEOPLE_POWER_KEY = 'people-power'
+
 USER_SERVICES = (
```

Comment:
Please use the same naming scheme as for the choices tuples above, for this one.